src/broker/broker.c

```diff
@@ -1879,7 +1879,7 @@ static int module_insmod_respond (flux_t *h, module_t *p)
     int rc;
     int errnum = 0;
     int status = module_get_status (p);
-    flux_msg_t *msg = module_pop_insmod (p);
+    const flux_msg_t *msg = module_pop_insmod (p);
```

Comment:
commit message typo "but we struct" ... "but we use struct ..." or just "but struct ..."